scripts/screener/screener.runner.ts

```diff
@@ -107,7 +111,13 @@ export async function screenerRunner(screenerConfig: ScreenerRunnerConfig) {
       : undefined,
   });
 
-  await notifyIntegration({ commit, url: checkUrl, status: 'in_progress', project: screenerConfig.projectRepo });
+  await notifyIntegration({
```

Comment:
notifyIntegration is also used below, if you address https://github.com/microsoft/fluentui/pull/24180/files#r935383853 you'll see what I'm talking about